biorbd_optim/limits/constraints.py

```diff
@@ -106,7 +106,7 @@ def non_slipping(
 
             mu = static_friction_coefficient
             for i in range(len(u)):
-                contact = nlp["contact_forces_func"](x[i], u[i], p)
+                contact = nlp.contact_forces_func(x[i], u[i], p)
```

Comment:
`contact_forces_func` should be somehow move next to `dynamics`  (or `dynamics_func`)

Reply:
Don't loose time on time, it is okay as is